### PR TITLE
Create color array to avoid similar colors and duplicates

### PR DESCRIPTION
**Pull request by thelaurafranklin**
Let me know if you don't like any of the colors.  Wasn't 100% on the gray scale and we don't really need 30 color options.

### CODE REVIEW

File: build/js/Game.js [hunk 1]
```diff
@@ -93,12 +93,17 @@ Game.prototype.assignRandomColors = function() {
   var cardbacks = document.querySelectorAll('.card-back');
   var color;
```
Comment (adnanusman) [inline build/js/Game.js:94]: I don't see this variable being used anywhere, if its not then maybe take it out :)

File: build/js/Game.js [hunk 2]
```diff
@@ -93,12 +93,17 @@ Game.prototype.assignRandomColors = function() {
...
 
-  // Add 6 random hexcodes to the randomColors array.
-  for(var i = 0; i < 6; i++) { 
-    color = this.generateRandomHexcode();
+  // array of colors to choose from
+  var colorOptions = ['#333', '#666', '#999', '#ccc', '#000033', '#000066', '#000099', '#0000cc', '#0000ff', '#003300', '#006600', '#009900', '#00cc00', '#00ff00', '#00ffcc', '#00ccff', '#00ffff', '#330000', '#660000', '#990000', '#cc0000', '#ff0000', '#660066', '#660099', '#9900ff', '#ccff00', '#ccffcc', '#99ff00', '#ff9900', '#ffcc33'];
```
Comment (adnanusman) [inline build/js/Game.js:97]: I like most of the colors.. there still seems to be a lot of similar colors though.

Maybe if we only put in only one light and one dark combination of a certain color?

**Review by adnanusman — COMMENTED** [review]
Just some minor changes

File: build/js/Game.js (hunk 1 above)
Comment (thelaurafranklin) [reply build/js/Game.js:94]: It is being used in the code you posted below...

File: build/js/Game.js (hunk 2 above)
Comment (thelaurafranklin) [reply build/js/Game.js:97]: I will make there be more variety

File: build/js/Game.js (hunk 1 above)
Comment (adnanusman) [reply build/js/Game.js:94]: do you mean color = this.generateRandomHexcode(); ? That was removed.

File: build/js/Game.js (hunk 2 above)
Comment (thelaurafranklin) [reply build/js/Game.js:97]: Improved color contrast

File: build/js/Game.js (hunk 1 above)
Comment (thelaurafranklin) [reply build/js/Game.js:94]: I see that now.  I accidentally just commented it out instead of deleting it, but it can be removed.

**Comment by adnanusman** [comment]
Thank you @flaura42 :)

**Pull request merged**